zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/steps/MetricsStep.java

```diff
@@ -31,25 +31,22 @@ public ActorFuture<Void> prepareTransition(
       // because any new meters will no longer be forwarded to the broker registry.
       partitionMeterRegistry.remove(brokerMeterRegistry);
       partitionMeterRegistry.close();
-      context.setPartitionMeterRegistry(null);
+      context.setPartitionTransitionMeterRegistry(null);
     }
     return context.getConcurrencyControl().createCompletedFuture();
   }
 
   @Override
   public ActorFuture<Void> transitionTo(
       final PartitionTransitionContext context, final long term, final Role targetRole) {
-    final var brokerRegistry = context.getBrokerMeterRegistry();
+    final var brokerRegistry = context.getPartitionStartupMeterRegistry();
```

Comment:
:question: I find a bit counter-intuitive that the "brokerRegistry" already has a partition tag registered in it. Has it always been like this?

Reply:
Oops, no, it should be renamed. I renamed the getter but not the variable :see_no_evil: